crates/rnote-ui/data/meson.build

```diff
@@ -377,6 +379,8 @@ rnote_ui_gresources_ui_files = files(
     'ui/style.css',
     'ui/unitentry.ui',
     'ui/workspacebrowser.ui',
+    'ui/workspacesbar/workspacerow.ui',
+    'ui/workspacesbar/workspacesbar.ui',
```

Comment:
I don't think this MR should need to touch this list. I ordered them so that folders are before files.

Reply:
unless you have a formatter for meson? I'd happily apply a standard formatting if I can run it myself (and maybe add it to the fmt check in CI)

Reply:
I'll format the meson files separately, so that this PR is not cluttered with these formatting changes.